playbooks/rds_monitoring_start_server.yaml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Prepare controller variables
+  import_playbook: prepare_variables.yaml
+  vars:
+    scenario_name: >
+      csm_controller
+      rds_monitoring
+
+- name: Prepare variables
+  hosts: localhost
+  vars_files:
+    - ./vars/csm_controller.yaml
+    - ./vars/rds_monitoring.yaml
```

Comment:
Var files are missing

Reply:
this files creates by prepare_variables.yaml